src/popup/router/components/TotalWalletAmount.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="total-amount">
+    <span>{{ totalAmount }}</span>
+    <span class="label">
+      {{ $t('total') }}
+    </span>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+
+export default {
+  subscriptions() {
+    return pick(this.$store.state.observables, ['balances']);
+  },
+  computed: {
+    ...mapGetters(['convertToCurrencyFormatted', 'accounts']),
+    totalAmount() {
+      if (!this.balances) return 0;
```

Comment:
https://github.com/aeternity/superhero-wallet/blob/67f25570223516b83792d9ffddc28e90daf484fa/src/store/plugins/observables.js#L66-L72

I don't think that this condition would work.